Libraries/Text/RCTTextFieldManager.m

```diff
@@ -39,8 +39,16 @@ - (BOOL)textField:(RCTTextField *)textField shouldChangeCharactersInRange:(NSRan
   }
 
   if (textField.maxLength == nil || [string isEqualToString:@"\n"]) {  // Make sure forms can be submitted via return
+    if (textField.autocorrectionType == UITextAutocorrectionTypeYes) {
```

Comment:
Here are the possible types:
https://developer.apple.com/reference/uikit/uitextautocorrectiontype?language=objc

Would changing the line above be what we want here?

    if (textField.autocorrectionType != UITextAutocorrectionTypeNo)